### DISCUSSION TITLE
Can't build against v1.6

### DISCUSSION THREAD

**Discussion by LukeAI** [microsoft/onnxruntime#6853] [Other Q&A]
Can't build minimal example against onnxruntime:

Ubuntu 18.04

download the [v1.6](https://github.com/microsoft/onnxruntime/releases/download/v1.6.0/onnxruntime-linux-x64-1.6.0.tgz) and extract to /opt/onnxruntime

make a cmakelists with:

```
include_directories(/opt/onnxruntime/include)

add_executable(sample sample.cpp)
target_link_libraries(sample PRIVATE onnxruntime)
```

[sample.cpp](https://github.com/microsoft/onnxruntime/blob/master/csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/CXX_Api_Sample.cpp)

now build with:
mkdir build && cd build && cmake .. && make

```
-- Configuring done
-- Generating done
-- Build files have been written to: /home/luke/projects/onnx_sample/build
Scanning dependencies of target sample
[ 50%] Building CXX object CMakeFiles/sample.dir/sample.cpp.o
[100%] Linking CXX executable sample
/usr/bin/ld: cannot find -lonnxruntime
collect2: error: ld returned 1 exit status
CMakeFiles/sample.dir/build.make:102: recipe for target 'sample' failed
make[2]: *** [sample] Error 1
CMakeFiles/Makefile2:94: recipe for target 'CMakeFiles/sample.dir/all' failed
make[1]: *** [CMakeFiles/sample.dir/all] Error 2
Makefile:102: recipe for target 'all' failed
make: *** [all] Error 2
```

**Comment by snnn** [comment]
Please run:

"make VERBOSE=1"

And see how it was linked. 

You need to add  "/opt/onnxruntime/lib" to the link directories. 

See: https://cmake.org/cmake/help/latest/command/target_link_directories.html
